pages/about.jsx

```diff
@@ -1,24 +1,44 @@
 import Image from 'next/image';
 import Link from 'next/link';
+import aboutPic1 from '/Users/rohangandhi/Downloads/BResearch/bresearch/images/aboutPic1.png';
+import aboutPic2 from '/Users/rohangandhi/Downloads/BResearch/bresearch/images/aboutPic2.png';
+import aboutPic3 from '/Users/rohangandhi/Downloads/BResearch/bresearch/images/aboutPic3.png';
+import logo from '/Users/rohangandhi/Downloads/BResearch/bresearch/images/logo.png';
```

Comment:
Use relative paths. Use the public folder if it works. See https://nextjs.org/docs/pages/building-your-application/optimizing/images#local-images